src/phoenix/server/app.py

```diff
@@ -250,38 +267,74 @@ async def __aexit__(self, *args: Any, **kwargs: Any) -> None:
         await self.stop()
 
     async def _run(self) -> None:
-        logger.info("Loading Trace Fixtures.")
-        await self._handle_tracing_fixtures()
-        logger.info("Finished loading fixtures. You may have to refresh.")
-
-    async def _handle_tracing_fixtures(self) -> None:
-        for fixture in self._tracing_fixtures:
-            trace_ds = await asyncio.get_running_loop().run_in_executor(
-                None, load_example_traces, fixture.name
+        """
+        Main entry point for Scaffolder.
+        Determines whether to load fixtures and handles them.
+        """
+        if await self._should_load_fixtures():
+            logger.info("Loading trace fixtures.")
+            await self._handle_tracing_fixtures()
+            logger.info("Finished loading fixtures.")
+        else:
+            logger.info("DB is not new, avoid loading demo fixtures.")
+
+    async def _should_load_fixtures(self) -> bool:
+        if self._force_fixture_ingestion:
+            return True
+
+        async with self._db() as session:
+            created_at = await session.scalar(
+                select(models.Project.created_at).where(models.Project.name == "default")
             )
 
-            fixture_spans, fixture_evals = await asyncio.get_running_loop().run_in_executor(
-                None,
-                reset_fixture_span_ids_and_timestamps,
-                (
-                    # Apply `encode` here because legacy jsonl files contains UUIDs as strings.
-                    # `encode` removes the hyphens in the UUIDs.
-                    decode_otlp_span(encode_span_to_otlp(span))
-                    for span in trace_ds.to_spans()
-                ),
-                get_evals_from_fixture(fixture.name),
-            )
-            project_name = fixture.project_name or fixture.name
-            logger.info(f"Loading '{project_name}' fixtures...")
-            for span in fixture_spans:
-                await self._queue_span(span, project_name)
-            for evaluation in fixture_evals:
-                await self._queue_evaluation(evaluation)
+        is_new_db = created_at is None or (
```

Comment:
if `default` is missing, e.g. someone manually deleted it, then it's probably not new?
```suggestion
        is_new_db = created_at and (
```

Reply:
Is not deletable AFAIK